features/aragon/aragonForm.tsx

```diff
@@ -0,0 +1,111 @@
+import { Block, Button, ToastError } from '@lidofinance/lido-ui';
+import { useAragonVote } from 'features/vesting';
+import { SelectVesting } from 'features/vesting/selectVesting';
+import { useCallback } from 'react';
+import { useForm } from 'react-hook-form';
+import { InputGroupStyled, InputNumber } from 'shared/ui';
+import { useEncodeAragonCalldata } from 'features/votingAdapter';
+import { Form } from './aragonFormStyles';
+import { stringToNumber } from 'shared/lib';
+import { useGetVoting } from './useAragon';
+
+type AragonFormData = {
+  voteId: string;
+  success: boolean;
+};
+
+const validateVoteId = (value: string) => {
+  const number = Number(value);
+  if (number.toString() !== value) {
+    return 'Must be a number';
+  }
+  if (number < 0) {
+    return 'Must be greater than zero';
+  }
+  return true;
+};
+
+export const AragonForm = () => {
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { isValid, errors },
+  } = useForm<AragonFormData>({ mode: 'onChange' });
+
+  const encodeCalldata = useEncodeAragonCalldata();
+  const aragonVote = useAragonVote();
+  const getVoting = useGetVoting();
+
+  const runTransaction = useCallback(
+    async (data: AragonFormData) => {
+      const { voteId, success } = data;
+
+      const vote = await getVoting(stringToNumber(voteId));
+      if (vote == null) {
+        ToastError(`Voting doesn't exists`);
+        return;
+      }
+      if (vote?.open === false) {
+        ToastError('Voting is closed');
+        return;
+      }
+
+      const callData = await encodeCalldata(stringToNumber(voteId), success);
```

Comment:
why you even need stringToNumber if you have parseInt/parseFloat?

Reply:
yeah, I don't need it, going to simplify